src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
After some extra debugging, here are my findings:

* The dialog is not being shown during the fist attempt of installing the package, but at the end of the installation, during ```Remote#configure_display_manager```
* The first attempt to install the package happens here [1], using ```Pkg::DoProvide```. For "libyui-qt6" it successes. For "libyui-qt" ```Pkg::DoProvide``` returns "the package is not available"

Should we use any alternative to ```DoProvide``` (one supporting provides) on ```SelectSystemPackages``` or should we do that work "by hand" beforehand on ```Packages.vnc_packages```

[1] https://github.com/yast/yast-packager/blob/master/src/modules/Packages.rb#L2161